go/rpcclientlib/obscuro_client.go

```diff
@@ -177,10 +177,14 @@ func (c *networkClient) decryptResponse(resultBlob interface{}) ([]byte, error)
 		// todo: remove this when we no longer support disabling viewing keys for testing
 		return unencrypted, nil
 	}
-	encryptedResult := common.Hex2Bytes(resultBlob.(string))
+	resultStr, ok := resultBlob.(string)
+	if !ok {
+		return nil, fmt.Errorf("expected hex string but result was not a string, it was (%t) %s", resultBlob, resultBlob)
```

Comment:
I think this might be clearer as `fmt.Errorf("expected hex string but result was of type %t instead, with value %s", resultBlob, resultBlob)`